opm/autodiff/SimulatorFullyImplicitBlackoil_impl.hpp

```diff
@@ -205,6 +207,7 @@ namespace Opm
                     boost::any_cast<const ParallelISTLInformation&>(solver_.parallelInformation());
                 // Only rank 0 does print to std::cout
                 terminal_output_= (info.communicator().rank()==0);
+                is_parallel_run_ = info.communicator().size();
```

Comment:
I assume that a communicator will have size 0 in the serial case then? I assumed it would be 1 (as in: we have a single thread running).